### ISSUE TITLE
[a11y] Icons in breadcrumbs are missing labels

### ISSUE THREAD

**Issue by jaredhenderson** [unimelb/pattern-lib#1651]
## Currently
Home and chevron icons in the breadcrumbs bar do not have an appropriate label.
https://study.unimelb.edu.au/sortsite-test

![Screen Shot 2021-06-03 at 3.19.36 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/493105d8-695b-429f-a10b-dc92162a7192)

## Proposed changes
- [x] add a label to home SVG of 'Home icon'
- [x] add a label to the chevron SVG of 'Chevron right'
- [x] hide the icons from being read by screenreaders
- [x] these changes need to be made in FAC as well if the breadcrumbs are implemented separately there

**Comment by gepifanio** [comment]
PR: https://github.com/unimelb/pattern-lib/pull/1655
